package.json

```diff
@@ -1,56 +1,58 @@
 {
-  "name" : "@scm-manager/scm-jenkins-plugin",
-  "private" : true,
-  "version" : "2.1.0-SNAPSHOT",
-  "license" : "MIT",
-  "main" : "./src/main/js/index.ts",
-  "scripts" : {
-    "build" : "ui-scripts plugin",
-    "watch" : "ui-scripts plugin-watch",
-    "postinstall" : "ui-plugins postinstall"
+  "name": "@scm-manager/scm-jenkins-plugin",
+  "private": true,
+  "version": "2.1.0-SNAPSHOT",
+  "license": "MIT",
+  "main": "./src/main/js/index.ts",
+  "scripts": {
+    "build": "ui-scripts plugin",
+    "watch": "ui-scripts plugin-watch",
+    "postinstall": "ui-plugins postinstall"
   },
-  "dependencies" : {
-    "@scm-manager/ui-plugins" : "2.0.0",
-    "@scm-manager/ui-components" : "^2.0.0",
-    "@scm-manager/ui-extensions" : "^2.0.0",
-    "classnames" : "^2.2.6",
-    "query-string" : "^5.0.1",
-    "react" : "^16.10.2",
-    "react-i18next" : "^10.13.1",
-    "react-redux" : "^5.0.7",
-    "react-router-dom" : "^5.1.2",
-    "redux" : "^4.0.0",
-    "styled-components" : "^4.4.0"
+  "dependencies": {
```

Comment:
Do we need the suffix in the @scm-manager dependencies or would a simple "^2.3.0" suffice ?

Reply:
This is auto generated.